paddle/fluid/pybind/op_function_generator.cc

```diff
@@ -561,6 +567,11 @@ int main(int argc, char* argv[]) {
     return -1;
   }
 
+#ifdef PADDLE_WITH_ASCEND
+  auto ascend_ptr = paddle::framework::AscendInstance::GetInstance();
+  ascend_ptr->InitGEForUT();
+#endif
```

Comment:
Not important to this PR, but I wonder why need this?

Reply:
weibao: for ut